TestingTravis/ViewController.swift

```diff
@@ -13,6 +13,8 @@ class ViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         // Do any additional setup after loading the view, typically from a nib.
+        
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Nakul97/TestingTravis/blob/2ab967952d0734b15797e4d70bbe1b3c3fb02e82/TestingTravis/ViewController.swift#L16"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>ViewController.swift:16</code></td>
    </tr>
  </tbody>
</table>